easybuild/easyconfigs/z/Zgoubi/Zgoubi-6.0.2-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'Zgoubi'
+version = '6.0.2'
+
+homepage = 'https://zgoubi.sourceforge.io'
+description = """Zgoubi is a ray-tracing code in use for charged particle beam dynamics simulations. 
+It can simulate beam dynamics in a large variety of machines and optical systems."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('GCC', '10.3.0'),
+]
+
+source_urls = ['https://sourceforge.net/projects/zgoubi/files/zgoubi/']
+sources = ['%(version)s/zgoubi-%(version)s.tar.bz2']
+checksums = ['0b0f3d28496edbf7be868c24b0c927b7f1bc88cc29df9c235d9b28dc933489e0']
+
+prebuildopts = 'cp Makefile_zgoubi_gfortran Makefile &&'
+prebuildopts += "sed -i '/guide/d' Makefile &&"
+
+buildopts = ' FFLAGS="$FFLAGS -fno-automatic -pedantic -cpp" CFLAGS="$CXXFLAGS -DGFORTRAN4"'
+
+files_to_copy = [(['zgoubi/zgoubi', 'zpop/zpop'], 'bin')]
```

Comment:
Maybe the `examples` subdirectory should also be copied?

Reply:
You mean, add to files_to_copy?
I will try, but for any case, how can I remove this one from pushed set?
I tried 
#eb --update-pr 14516  --pr-commit-msg  "Removed zgoubi"  :zgoubi-6.0.2-GCCcore-10.3.0.eb
But it gave an error

Reply:
The filename in the command is case-sensitive, so you should try with capital `Z`:

```
eb --update-pr 14516 --pr-commit-msg "Removed zgoubi" :Zgoubi-6.0.2-GCCcore-10.3.0.eb
```

Reply:
I'm not sure why you need to remove it all though, since there's only one Zgoubi easyconfig file in this PR.

Note that the existing easyconfig is updated if you upload a changed easyconfig via `--update-pr`.